app/src/main/java/com/jvoye/tasky/core/data/networking/mappers/DtoMappers.kt

```diff
@@ -112,40 +153,17 @@ fun TaskyItem.toCreateReminderRequest(): CreateReminderRequest {
         id = id,
         title = title,
         description = description,
-        time = time.toString(),
-        remindAt = remindAt.toString(),
-        updatedAt = Clock.System.now().toLocalDateTime(TimeZone.UTC).toString()
+        time = time.toInstant(TimeZone.UTC).toString(),
+        remindAt = remindAt.toInstant(TimeZone.UTC).toString(),
+        updatedAt = getUpdatedAtTime()
     )
 }
 
-
-/*
-RESPONSE TASK
-{
-    "id": "f47ac10b-58cc-4372-a567-0e02b2c3d479", (String)
-    "title": "Complete API documentation", (String)
-    "description": "Write comprehensive API docs", (String)
-    "time": "2024-01-15T14:00:00Z", (String)
-    "remindAt": "2024-01-15T13:45:00Z", (String)
-    "isDone": true (Boolean)
-}
-
-RESPONSE REMINDER
-{
- "id": "f47ac10b-58cc-4372-a567-0e02b2c3d479", (String)
- "title": "Doctor Appointment", (String)
- "description": "Annual checkup", (String)
- "time": "2024-01-15T16:00:00Z", (String)
- "remindAt": "2024-01-15T15:30:00Z" (String)
-}
-
-RESPONSE FULL AGENDA
-{
-"events": [...events], (List<Event>)
-"tasks": [...tasks], (List<Task>)
-"reminders": [...reminders] (List<Reminder>)
+@OptIn(ExperimentalTime::class)
+private fun getUpdatedAtTime(): String {
```

Comment:
Wouldn’t it make more sense for the `updatedAt` field to come from the server, rather than being generated locally with `getUpdatedAtTime()`?